portal/src/main/webapp/js/src/data_download.js

```diff
@@ -221,7 +221,7 @@ var DataDownloadTab = (function() {
             }
             
             // Hide download file links for multi-studies.
-            if (_processedProfiles.length > 1) {
+            if (window.QuerySession.isVirtualStudy) {
```

Comment:
@jjgao please confirm the logic here.  Virtual study will be true if study is virtual cohort OR if multi-study.